photon-api/src/main/scala/com/linkedin/photon/ml/model/MatrixFactorizationModel.scala

```diff
@@ -220,6 +252,55 @@ object MatrixFactorizationModel {
         }
       }
 
-    new KeyValueScore(scores)
+    new ModelDataScores(scores)
+  }
+
+  /**
+   * Score the given GAME data points with the row and column latent factors.
+   *
+   * @param dataPoints The GAME data points
+   * @param rowEffectType The type of row effect used to retrieve row effect id from each GAME data point
+   * @param colEffectType The type of column effect used to retrieve column effect id from each GAME data point
+   * @param rowLatentFactors The row latent factors
+   * @param colLatentFactors The col latent factors
+   * @return The computed scores
+   */
+  private def scoreLite(
```

Comment:
`scoreForCoordinateDescent` ? - it would have the merit of telling right away where it is used, so it would be good for orientation in the code?